### PR TITLE
feature: implement offline push api

### PR DESCRIPTION
**Pull request by wfxr**
TODO:
- [x] implement offline push api
- [x] implement offline create table api
- [x] Add unit tests

### CODE REVIEW

File: internal/database/offline/sqlutil/push.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/oom-ai/oomstore/internal/database/dbutil"
+	"github.com/oom-ai/oomstore/internal/database/offline"
+)
+
+func cdcTableName(groupID int, revision int64) string {
```
Comment (jinghancc) [inline internal/database/offline/sqlutil/push.go:12]: replace `OfflineStreamCdcTableName`?

**Review by YikSanChan — APPROVED** [review]
lgtm

**Review by lianxmfor — APPROVED** [review]
lgtm

**Review by YikSanChan — APPROVED** [review]
lgtm

**Review by jinghancc — APPROVED** [review]
LGTM

**Pull request merged**